script_runner/tests/test_ctx_wrapper.py

```diff
@@ -65,7 +78,15 @@ def _run(self, script,
 
         self.script_path = self._create_script(script)
         process = process or {}
-        process.update({'ctx_proxy_type': 'unix'})
```

Comment:
no reason to hardcode that; autodetection will make it work on both unix (using a unix socket by default) and windows